### ISSUE TITLE
Language changing from en-GB to en-US

### ISSUE THREAD

**Issue by fab7jj** [fingerprintjs/fingerprintjs#571]
- What causes the fingerprint to change?

I was testing and noticed that sometimes on the same environment, a different device Id was being generated. Thus I logged in the console the components values and the only two things that where changing are the language and canvas. I am fine with excluding the canvas to have a consistent device Id since without it my app is practically useless, but for the language I can't understand why something so simple is not consistent, sometimes its value was en-GB, other times en-US. Any insight please?

- What device and browser are you using?

OnePlus smartphone running android 10.3.6 using chrome browser version 86.0.4240.75

- What version of fingerprintjs are you using? 

v2.1.4

**Comment by Finesse** [comment]
What happens before the language changes? Do you restart the browser, change device settings, etc?

**Comment by fab7jj** [comment]
Nothing out of the ordinary to be honest with you, just randomly when I refresh the tab of my web app in the browser I get a different device id and I already have the below lot excluded: 

```
const fingerprintOptions = {
            excludes: {
                //defaults
                enumerateDevices: true,
                pixelRatio: true,
                doNotTrack: true,
                fontsFlash: true,
                //custom
                adBlock: true,
                plugins: true,
                sessionStorage: true,
                localStorage: true,
                indexedDb: true,
                openDatabase: true,
                addBehavior: true,
                fonts: true,
                audio: true
            }
        };
```

**Comment by Valve** [comment]
@Finesse is this related to Chrome 86 languages used in translation issue we fixed in Pro?

**Comment by Finesse** [comment]
@Valve No, version 2 isn't affected by the Chrome 86 changes because it uses only the first language from `navigator.languages` which is always equal to `navigator.language`.

@fab7jj Please get a couple of changing fingerprints at https://jlryy.csb.app/ and paste them here (including the JSONs with component lists). Also, since version 2 is outdated, please try to [upgrade to version 3](https://github.com/fingerprintjs/fingerprintjs/blob/master/migrating_v2_v3.md) and tell whether the problem is presented there.

**Comment by Finesse** [comment]
Looks like the fingerprint change is caused by using another Chrome profile. Each Chrome profile has its own settings, that includes language and fonts (fonts affect the canvas). The canvas issue has been fixed d8370403175c33d2f9542b172f9c9ea9fa3c17bd. But I doubt that separate Chrome profile should be treated as the same visitor, because Chrome settings contain much entropy, therefore we can't discard the entropy sources based on the settings.

**Comment by Finesse** [comment]
We've decided that guest mode is a separate but stable browser instance. So, we shouldn't discard signals in favor of guest mode.
